### PR TITLE
Update opi_video_publishing_policy.md

### PR DESCRIPTION
**Pull request by jlwhite**

### CODE REVIEW

File: opi_video_publishing_policy.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Open Programmable Infrastructure (OPI) Video and Presentation approval Process
+
+This policy covers the review of technical content for public presentations and Videos about the Open Programmable Infrastructure (OPI) project. The intent is to have a consistent, accurate, and uniform representation of OPI as well as to assist member companies and individuals.
+
+- Scope
+  - This policy will cover videos, presentations, or documents with OPI technical content that for use in public forums such as industry conferences, academic conferences, or webinars.
+  - Company internal presentations or company customer facing presentations that include OPI information or are about OPI are not covered by this policy
+  - To facilitate reuse a set of common slides and content will be approved jointly with the OPI Outreach committee. This will streamline the process of creating and delivering presentions about OPI as reuse of already approved content will not require a new review.
+  - OPI TSC, Subproject, and Working Group meetin minutes, videos, and recordings do not need approval to post throught https://lists.opiproject.org/ or https://github.com/opiproject.
```
Comment (mestery) [inline opi_video_publishing_policy.md]: `meeting` instead of `meetin` and `through` instead of `throught`.

Comment (mestery) [reply opi_video_publishing_policy.md]: Also, those links need to be wrapped in `[]` to pass the markdown linter.

File: opi_video_publishing_policy.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+<br><br>
+- Process
+  - Upload the content to the OPI google docs presentation area
+  - Request for review by the TSC will be via the TSC mailing list or in person during a TSC meeting
+  - Approval of new content will be voted on by the TSC at the TSC meeting following the request for review
+  - Approved presentations will be posted to opiproject github
+  - Approved videos may be posted to the OPI YouTube Channel
+<br><br>
+- YouTube channel 
+  - management and maintenance will be by the outreach committee
```
Comment (mestery) [inline opi_video_publishing_policy.md]: Add `with oversight by the TSC`? Or maybe that is assumed.

File: opi_video_publishing_policy.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+This policy covers the review of technical content for public presentations and Videos about the Open Programmable Infrastructure (OPI) project. The intent is to have a consistent, accurate, and uniform representation of OPI as well as to assist member companies and individuals.
+
+- Scope
+  - This policy will cover videos, presentations, or documents with OPI technical content that for use in public forums such as industry conferences, academic conferences, or webinars.
+  - Company internal presentations or company customer facing presentations that include OPI information or are about OPI are not covered by this policy
+  - To facilitate reuse a set of common slides and content will be approved jointly with the OPI Outreach committee. This will streamline the process of creating and delivering presentions about OPI as reuse of already approved content will not require a new review.
+  - OPI TSC, Subproject, and Working Group meetin minutes, videos, and recordings do not need approval to post throught https://lists.opiproject.org/ or https://github.com/opiproject.
+<br><br>
+- Process
+  - Upload the content to the OPI google docs presentation area
```
Comment (glimchb) [inline opi_video_publishing_policy.md]: will be helpful to have a link to google drive area

File: opi_video_publishing_policy.md (hunk 2 above)
Comment (glimchb) [reply opi_video_publishing_policy.md]: will be helpful to have a link to youtube channel

File: opi_video_publishing_policy.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Open Programmable Infrastructure (OPI) Video and Presentation Approval Process
+
+This policy covers the review of technical content for public presentations and Videos about the Open Programmable Infrastructure (OPI) project. The intent is to have a consistent, accurate, and uniform representation of OPI as well as to assist member companies and individuals.
+
+- Scope
+  - This policy will cover videos, presentations, or documents with OPI technical content that for use in public forums such as industry conferences, academic conferences, or webinars.
```
Comment (timmichels) [inline opi_video_publishing_policy.md]: Can this also say "where OPI is being officially represented"

**Comment by venkat-pullela-keys** [comment]
This "the process of creating and delivering presentations about OPI as reuse of already approved content will not require a new review" needs to move from scope to process.

**Comment by venkat-pullela-keys** [comment]
Approved slides should be presentation ready, and not a template that needs to be filled with content to make it presentable.

**Review by glimchb — APPROVED** [review]
LGTM

**Review by seroyer — COMMENTED** [review]
Please address the linter failures and @jlwhite please address the DCO failure (commits need signoffs)

File: opi_video_publishing_policy.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Open Programmable Infrastructure (OPI) Video and Presentation Approval Process
+
+This policy covers the review of technical content for public presentations and Videos about the Open Programmable Infrastructure (OPI) project. The intent is to have a consistent, accurate, and uniform representation of OPI as well as to assist member companies and individuals not to restrict public discussion of about OPI.
```
Comment (krmurph) [inline opi_video_publishing_policy.md]:
```suggestion
This policy covers the review of technical content for public presentations and Videos about the Open Programmable Infrastructure (OPI) project. The intent is to have a consistent, accurate, and uniform representation of OPI as well as to assist member companies and individuals not to restrict public discussion about OPI.
```

File: opi_video_publishing_policy.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  - Company internal presentations or company customer facing presentations that include OPI information or are about OPI are not covered by this policy.
+  - To facilitate reuse a set of common slides and content will be approved jointly with the OPI Outreach committee. This will streamline the process of creating and delivering presentions about OPI.
+  - Meeting minutes, videos, and recordings for OPI TSC, Subproject, and Working Group do not need approval to post through [https://lists.opiproject.org/] or [https://github.com/opiproject].
+<br><br>
+- Technical Content Approval Process
+  - Upload the new technical content to the OPI google docs 'under review' presentation area
+    - Reuse of already approved content will not require a review.
+      - Reuse of approved existing content in OPI public presentations in a different aggrangement, sequence, subset, or with a differnt visual arrangement does not require a new approval.
```
Comment (krmurph) [inline opi_video_publishing_policy.md]:
```suggestion
      - Reuse of approved existing content in OPI public presentations in a different arrangement, sequence, subset, or with a different visual arrangement does not require a new approval.
```

File: opi_video_publishing_policy.md (hunk 5 above)
Comment (jlwhite) [reply opi_video_publishing_policy.md]: fixed

File: opi_video_publishing_policy.md (hunk 6 above)
Comment (jlwhite) [reply opi_video_publishing_policy.md]: fixed

**Comment by krmurph** [comment]
LGTM

**Comment by jlwhite** [comment]
Updated version committed and pushed. Fixed HTML links and formatting and spelling errors.

**Review by mestery — CHANGES_REQUESTED** [review]
This has merge commits. It would be best to either do a new one or get rid of those merge commits and do a rebase against main.

**Review by timmichels — APPROVED** [review]
Reads very clean now.  And the links work!

**Comment by glimchb** [comment]
switched to #37

**Comment by jlwhite** [comment]
updated policy in publishing_policy.md with new pull request see #37

**Pull request closed without merging**